index.bs

```diff
@@ -1765,6 +1792,39 @@ a [=moment=] |sourceTime|, and a [=duration=] |eventReportWindow|:
     1. Set |lastEnd| to |lastEnd| + |deadline|.
 1. Return |windows|.
 
+To <dfn>parse report windows</dfn> given a [=map=] |map|, a [=moment=] |sourceTime|,
+and an [=report window list=] |default|:
+
+1. [=Assert=]: |default| is not [=list/is empty|empty=].
+1. Let |defaultEndDuration| be the [=duration from=] |sourceTime| and the [=report window/end=] of the last element of |default|.
```

Comment:
Let's move this step closer to where this variable is first used, i.e. before iterating over `|endDurations|`.